tag_locations:
  [
    {id: 1, x: 0.474283, y: -1.460830, z: 0.100586, qx: -0.034613, qy: -0.664205, qz: -0.746627, qw: 0.013439},
    {id: 2, x: 3.674, y: 6.360, z: 0.153, qx: 0.670, qy: -0.023, qz: -0.013, qw: 0.742},
    {id: 4, x: 0.526227, y: 2.211401, z: -0.030423, qx: 0.668465, qy: -0.010406, qz: -0.009038, qw: 0.743616},
    {id: 7, x: 1.191, y: 0.568, z: 0.086, qx: 0.523, qy: -0.390, qz: -0.455, qw: 0.606},
    # {id: 7, x: 0.0, y: 0.0, z: 0.0, qx: 0.0, qy: 0.0, qz: 0.0, qw: 1.0},
    {id: 8, x: 0.873, y: -1.957, z: 0.086, qx: 0.055, qy: 0.658, qz: 0.749, qw: 0.063}
  ]
max_detection_dist: 1.5     # < 1.5
max_linear_vel_x: 1.0       # max: 1.0, < 1.0
max_angular_vel_z: 0.6      # max: 2.0, < 0.6
xy_tolerance: 0.3           # > 0.3
yaw_tolerance: 0.3          # > 0.3
# update_frequency: 1.0       # < 10hz, default 1hz
skip_data_num: 5            # 15 = 1s
update_interval: 45         # 15 = 1s
